lte/gateway/c/core/oai/tasks/amf/amf_app_handler.cpp

```diff
@@ -617,13 +616,13 @@ void amf_app_handle_pdu_session_response(
       OAILOG_ERROR(
           LOG_AMF_APP, "pdu session  not found for session_id = %u\n",
           pdu_session_resp->pdu_session_id);
-      return;
+      return rc;
     }
     ue_id = ue_context->amf_ue_ngap_id;
   } else {
     OAILOG_ERROR(
         LOG_AMF_APP, "ue context not found for the imsi=%lu\n", imsi64);
-    return;
+    return rc;
```

Comment:
Better to be explicit
```suggestion
    return RETURNerror;
```

Reply:
addressed review comment